test/test_failover.rb

```diff
@@ -132,4 +132,45 @@
       end
     end
   end
+
+  describe "broken sockets" do
+
+    it 'get should replace the broken socket' do
+      memcached(29199) do |old_dc|
+        # Open a socket
+        assert_equal(nil, old_dc.get('foo'))
```

Comment:
@ccoffey This should be an `assert_nil` - the use of `assert_equal` here generates a deprecation warning.